src/index.js

```diff
@@ -43,8 +43,8 @@ function EvolvClient(options) {
 
   const store = options.store || new Store(options);
   const context = options.context || new Context(store);
-  const contextBeacon = options.analytics ? new Beacon(options.endpoint + '/' + options.environment + '/data', context) : null;
-  const eventBeacon = options.beacon || new Beacon(options.endpoint + '/' + options.environment + '/events', context);
+  const contextBeacon = options.analytics ? new Beacon(options.endpoint + '/' + options.environment + '/data', context, options.blockEvents) : null;
```

Comment:
Block is not entirely precise based on my reading. Maybe `buffer`?
```suggestion
  const contextBeacon = options.analytics ? new Beacon(options.endpoint + '/' + options.environment + '/data', context, options.bufferEvents) : null;
```